### ISSUE TITLE
Prefix for KV storage ?

### ISSUE THREAD

**Issue by noncapturesen** [iRedScarf/onedrive-index#15]
i don't know clearly about Prefix for KV storage and should be telegram channel for next update and Questions and Issue.

i want to show all share content in one Onedrive account ? 
Is it possible ?
i am testing with multiple onedrive accounts but i have face an issue with KV storage ?

**Comment by iRedScarf** [comment]
Yes, you can display all shared content in a OneDrive account, you just need to set **`BASE_DIRECTORY`** to **`/`**, which means you are sharing the **root directory** of this OneDrive.

**`KV_PREFIX`** (Key-Value Prefix) is a method used in key-value storage systems like **`Redis`** to avoid key conflicts. In your case, if you plan to deploy multiple OneDrive-Index instances, and all of them are **using the same Redis database**, setting different KV_PREFIX values for each instance is crucial.

How this works:

1. **Key-Value Storage System:** Redis is a key-value storage system that allows you to store and retrieve data using unique keys.

2. **Key Conflicts**: If multiple applications or instances use the same Redis database and they store data using the same keys, there could be an overwriting or mixing of data, leading to "key conflicts".

3. **Using KV_PREFIX:** By specifying a unique KV_PREFIX for each OneDrive-Index instance, you ensure that the keys created by each instance are unique. For example, if one instance has a KV_PREFIX of "index1" and another has "index2", even if they store a key with the same name (like "user_data"), these keys would actually become "index1_user_data" and "index2_user_data" in Redis, thereby avoiding conflicts.
For instance, without using KV_PREFIX, two instances might both attempt to store a key named "user_data", leading to the data from the first instance being overwritten by the second. But with KV_PREFIX, they would store as "index1_user_data" and "index2_user_data", not interfering with each other.

This method is particularly useful in cloud services and multi-tenant applications where multiple independent instances need to share the same database resource but maintain data isolation.

**Comment by noncapturesen** [comment]
root directory does not show share folder sir.
IN case , i have 5 one drive accounts which all have 25 TB storage .
i wanna show all of my one drive account in one domain sir like GoogelShareDrives on Gindex or GOIndex.
Is it possible?

**Comment by iRedScarf** [comment]
I think I understand your idea. You want to share the contents of multiple accounts simultaneously on a OneDrive-Index website. 

This is not yet possible in the current version of OneDrive-Index, which can only display the contents of one OneDrive account. 

Of course, this functionality is feasible in code, but I am currently occupied with other matters and do not have plans to add this feature at the moment.
